genkit-tools/src/commands/eval-flow.ts

```diff
@@ -178,25 +185,31 @@ export const evalFlowRun = new Command('eval:flow')
     }
   );
 
-async function readInputs(data: string, filePath: string): Promise<any[]> {
+async function readInputs(
+  data: string,
+  filePath: string
+): Promise<EvalFlowInput> {
   const parsedData = JSON.parse(
     data ? data : await readFile(filePath!, 'utf8')
   );
   if (Array.isArray(parsedData)) {
     return parsedData as any[];
   }
 
-  return [parsedData];
+  return EvalFlowInputSchema.parse(parsedData);
```

Comment:
Consider showing a easy to read error message if the parse fails. I'm not sure with the zod error message is, but I assume it may be hard for folks to understand exactly what caused the failure.

Reply:
Done.